src/Internal.AspNetCore.Sdk/buildMultiTargeting/ApiCheck.targets

```diff
@@ -13,6 +13,15 @@
       RemoveProperties="TargetFrameworks" />
   </Target>
 
+  <Target
+      Name="Replace-ApiCheck-Baselines" >
+
+    <MSBuild Projects="$(MSBuildProjectFullPath)"
+      Targets="Replace-ApiCheck-Baselines"
+      Properties="TargetFramework=%(_TargetFrameworks.Identity)"
```

Comment:
You need to create the _TargetFrameworks item group. I may not be defined already.
```
     <ItemGroup>
       <_TargetFrameworks Remove="@(_TargetFrameworks)" />
       <_TargetFrameworks Include="$(TargetFrameworks)" />
     </ItemGroup>
```